contributors/devel/on-call-build-cop.md

```diff
@@ -7,9 +7,9 @@
 
 ### Traffic sources and responsibilities
 
-* GitHub Kubernetes [issues](https://github.com/kubernetes/kubernetes/issues)
-and [pulls](https://github.com/kubernetes/kubernetes/pulls): Your job is to be
-the first responder to all new issues and PRs. If you are not equipped to do
+* GitHub Kubernetes [issues](https://github.com/kubernetes/kubernetes/issues):
+Your job is to be
+the first responder to all new issues. If you are not equipped to do
```

Comment:
@grodrigues3 are issues auto-triaged at the moment? Did we change that policy? Is this line still accurate?